README.md

```diff
@@ -1 +1,103 @@
-# Hello-microverse
\ No newline at end of file
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
I suggest that you remove this logo and customize this template with info related to your project this will make it look professional.